### PR TITLE
Added the error detail while creating new course

### PR DESCRIPTION
**Pull request by iqbalcodes6602**
# Description

adding the exact error message if you do not select the school from the suggestion list for schools while creating a new school.

image

Fixes: #7470 

# Comments

Add any comments for the reviewers (e.g., indicating the beginning of the revision, something where the reviewer needs to pay special attention, etc.). If  there are no extra comments this section can be deleted.

# Checklist:

- [ ] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [ ] The tests were executed and all of them passed.
- [ ] If you are creating a feature, the new tests were added.
- [ ] If the change is large (> 200 lines), this PR was split into various Pull Requests. It's preferred to create one PR for changes in controllers + unit tests in PHPUnit,  and then another Pull Request for UI + tests in Jest, Cypress or both.

### CODE REVIEW

File: frontend/www/js/omegaup/course/new.ts [hunk 1]
```diff
@@ -49,17 +49,19 @@ OmegaUp.on('ready', () => {
...
-                    accept(data.school_id);
+                    resolve(data.school_id);
                   })
-                  .catch(ui.apiError);
+                  .catch((error) => {
+                    ui.apiError({ error: error.message });
+                  });
               } else {
-                accept(null);
+                reject(new Error('No school selected'));
```
Comment (pabo99) [inline frontend/www/js/omegaup/course/new.ts:64]: Could you please add this error string to the translation files?

You need to add the translation for the following languages:

- EN `frontend/templates/en.lang`
- ES `frontend/templates/es.lang`
- PT `frontend/templates/pt.lang`

**Review by pabo99 — COMMENTED** [review]
Great job!

Just one comment and this PR is okay for me

**Review by pabo99 — APPROVED** [review]
LGTM!

**Review by heduenas — APPROVED** [review]

**Comment by pabo99** [comment]
@iqbalcodes6602 could you please update this PR, once updated, it will be merged into main branch

**Pull request merged**